mso/utils.go

```diff
@@ -398,3 +400,32 @@ func GetPolicyByName(cont *container.Container, policyName string, templateEleme
 
 	return nil, fmt.Errorf("Policy name %s not found", policyName)
 }
+
+func testAccVerifyKeyValue(resourceAttrsMap *map[string]string, resourceAttrRootkey, stateKey, stateValue string) {
+	stateKeySplit := strings.Split(stateKey, ".")
+	for inputKey, inputValue := range *resourceAttrsMap {
+		if strings.Contains(stateKey, resourceAttrRootkey) && stateKeySplit[len(stateKeySplit)-1] == inputKey && (stateValue == inputValue || (inputValue == "reference" && stateValue != "")) {
+			delete(*resourceAttrsMap, inputKey)
+			break
+		}
+	}
+}
+
+func customTestCheckResourceAttr(resourceName, resourceAttrRootkey string, resourceAttrsMap map[string]string) resource.TestCheckFunc {
```

Comment:
Can we also change the name of the function indicating that it is used for type set?

Reply:
Done